### PR TITLE
Reskin/sidebar

### PR DESCRIPTION
**Pull request by adamgall**
Updates the left navigation sidebar

- [x] Hug bottom group of icons to bottom of viewport, with some buffer space.
- [x] Center the top group of icons in the empty space between Header and bottom icon group.
- [x] Make sure to give buffer space between two icon groups when viewport is short, and button groups bump up next to each other.
- [x] Give padding between left of viewport and beginning of button groups
- [x] Use new Phosphor icons
- [x] Updated background color for button groups
- [x] Updated border color for button groups
- [x] Updated border radius for button groups
- [x] Nuances in the bottom button group, for the language selector border
- [x] New expanded view of button groups on hover
- [x] Persistent expansion when viewport width > 1600px
- [x] Mobile menu

Some Screenshots:

Screenshot 2024-04-29 at 5 09 36 PM

Screenshot 2024-04-29 at 5 09 49 PM

Screenshot 2024-04-29 at 5 11 29 PM

Screenshot 2024-04-29 at 5 11 50 PM

Screenshot 2024-04-29 at 5 10 00 PM

Screenshot 2024-04-29 at 5 10 24 PM

Screenshot 2024-04-29 at 5 10 33 PM

### CODE REVIEW

File: src/components/ui/page/Navigation/NavigationLink.tsx [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-import { NavigationTooltip } from './NavigationTooltip';
+import { Link } from 'react-router-dom';
 
 interface INavigationLink {
   href: string;
   labelKey: string;
-  tooltipKey?: string;
   testId: string;
-  Icon: ComponentWithAs<'svg', IconProps>;
+  Icon: JSX.Element;
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/page/Navigation/NavigationLink.tsx:9]: I'll use `ReactElement`, some light research suggests that's the best option. Thanks for the callouts.

File: src/components/ui/page/Navigation/NavigationLink.tsx [hunk 2]
```diff
@@ -19,56 +16,34 @@ export function NavigationLink({
...
-      color: isActive ? 'gold.500' : 'inherit',
-      _hover: {
-        color: 'gold.500-hover',
-      },
-    };
-  }, [isActive]);
 
   return (
-    <NavigationTooltip label={t(tooltipTranslationKey)}>
+    <Box py={3}>
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/page/Navigation/NavigationLink.tsx:26]: Yes, probably. I did notice they were becoming very similar. I'll take a stab at combining them.

File: src/components/ui/page/Navigation/NavigationLinks.tsx [hunk 3]
```diff
@@ -1,22 +1,82 @@
...
+        borderTopWidth={1}
+        borderLeftWidth={1}
+        borderRightWidth={1}
+      >
+        <NavigationExternalLink
+          href={URL_FAQ}
+          ariaLabelKey="ariaLabelFAQ"
+          tooltipKey="faq"
+          testId="navigationExternal-faq"
+          Icon={<Question size={24} />}
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/page/Navigation/NavigationLinks.tsx:44]: I could use another pair of eyes on this, it's harder for me to implement than I was expecting.

File: src/components/ui/page/Navigation/NavigationLinks.tsx [hunk 4]
```diff
@@ -27,92 +87,68 @@ export function NavigationLinks({
...
-            my={8}
+        <Box marginY="auto">
+          <Box
+            maxWidth={{ base: 12, '3xl': '100%' }}
+            _hover={{ maxWidth: '100%' }}
+            transitionDuration="0.5s"
+            mt={12}
+            mb={3}
+            bg="neutral-2"
+            pl="11px"
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/page/Navigation/NavigationLinks.tsx:107]: Oh man you're telling me. Let's go over this in realtime too.

File: src/components/ui/page/Navigation/NavigationLinks.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/page/Navigation/NavigationLinks.tsx:44]: Fixed, thanks for your help!

**Comment by adamgall** [comment]
@DarksightKellar @mudrila I've addressed your comments, and marked this PR ready for review

**Review by DarksightKellar — APPROVED** [review]

File: src/components/ui/page/Header/index.tsx [hunk 5]
```diff
@@ -66,39 +64,54 @@ function HeaderLogo() {
             size="full"
             isFullHeight
           >
-            <DrawerOverlay />
             <DrawerContent
-              bg="chocolate.900"
+              bg="#221D25D6"
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/page/Header/index.tsx:68]: Correct, this color includes an alpha channel. We have some "What Alpha" and "Black Alpha" colors defined in `decent-ui`, but not this neutral color. I've added a comment here in the codebase.

File: src/components/ui/page/Navigation/NavigationLink.tsx [hunk 6]
```diff
@@ -1,74 +1,88 @@
...
+}: {
   labelKey: string;
-  tooltipKey?: string;
-  testId: string;
-  Icon: ComponentWithAs<'svg', IconProps>;
-  target?: string;
-  rel?: string;
-  closeDrawer?: () => void;
+  NavigationIcon: Icon;
+  t: TFunction<'navigation', undefined>;
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/page/Navigation/NavigationLink.tsx:14]: Two things:

1. I meant for line 14 to be more generic than that. It should actually just be
   ```ts
   t: TFunction;
   ```
   I just pushed an update.
2. I did it like this just to cut down on the number of `useTranslation` invocations happening in this file (to just one). Everything in this file is using the `navigation` translation namespace, so invoking the hook once and then prop drilling the result seemed right to me, for DRY reasons.

File: src/components/ui/page/Navigation/NavigationLink.tsx [hunk 7]
```diff
@@ -1,74 +1,88 @@
...
+  if (scope === 'internal') {
+    return (
       <Link
         data-testid={testId}
-        aria-label={t(tooltipTranslationKey)}
+        aria-label={t(labelKey)}
         to={href}
-        {...rest}
         onClick={closeDrawer}
+        style={{ display: 'block', paddingTop: '0.75rem', paddingBottom: '0.75rem' }}
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/page/Navigation/NavigationLink.tsx:62]: I might need your help with this one... The `Link` component here is from `react-router-dom`, not `chakra-ui`, and so doesn't have any `paddingTop` etc properties on it. I don't really know how else to add styles to this (I want to literally make the rendered anchor tag bigger) without either doing these inline styles, or defining a CSS class somewhere.

File: src/components/ui/page/Navigation/NavigationLink.tsx [hunk 8]
```diff
@@ -1,74 +1,88 @@
...
+    return (
+      <a
+        data-testid={testId}
+        aria-label={t(labelKey)}
+        href={href}
+        onClick={closeDrawer}
+        {...rest}
+        target="_blank"
+        rel="noreferrer noopener"
+        style={{ display: 'block', paddingTop: '0.75rem', paddingBottom: '0.75rem' }}
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/components/ui/page/Navigation/NavigationLink.tsx:80]: Anchor tags seem to not be `block` by default.

---

```ts
style={{ display: 'block', paddingTop: '0.75rem', paddingBottom: '0.75rem' }}
```
Screenshot 2024-04-30 at 9 19 45 AM

---

```ts
style={{ paddingTop: '0.75rem', paddingBottom: '0.75rem' }}
```
Screenshot 2024-04-30 at 9 20 19 AM

---

And can you help me with whatever it is you're suggesting to not use inline styles? I guess I'm not sure what approach you're suggesting.

**Review by nicolaus-sherrill — CHANGES_REQUESTED** [review]
This is looking great! 

I was having some weird issues with the component when I first pulled up the deploy preview (cursor glitching, needing to click to expand) but was unable to recreate once I reloaded and left the page open for a bit.

I don't see the hover, pressed, or active states active here yet, captured in this screen shot and recording.

Screenshot 2024-04-30 at 9 13 23 AM

https://github.com/decentdao/fractal-interface/assets/62394931/8cf95c68-cf03-484e-adaa-5a43588b58b4

**Comment by adamgall** [comment]
> This is looking great!
> 
> I was having some weird issues with the component when I first pulled up the deploy preview (cursor glitching, needing to click to expand) but was unable to recreate once I reloaded and left the page open for a bit.
> 
> I don't see the hover, pressed, or active states active here yet, captured in this screen shot and recording.
> 
> Screenshot 2024-04-30 at 9 13 23 AM
> 
>  Screen.Recording.2024-04-30.at.9.06.56.AM.mov

@nicolaus-sherrill sorry, where is that figma showing the hover and active states?

**Pull request merged**